src/openrct2/object/ObjectManager.cpp

```diff
@@ -120,14 +119,17 @@ class ObjectManager final : public IObjectManager
         auto requiredObjects = GetRequiredObjects(entries, count);
 
         // Load the required objects
-        size_t numNewLoadedObjects = 0;
-        auto loadedObjects = LoadObjects(requiredObjects, &numNewLoadedObjects);
+        {
+            auto loadedObjects = LoadObjects(requiredObjects);
```

Comment:
why even have the intermediate? saves having to write std::move

Reply:
I removed also the setter, don't see a reason to have it.